### PR TITLE
ampache: update to 4.2.3

### PR DESCRIPTION
**Pull request by aurieh**
Was tragically outdated with several CVEs to its name. This package is likely abandoned - I wouldn't mind taking ownership of it as I use the software myself.

### CODE REVIEW

File: srcpkgs/ampache/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'ampache'
 pkgname=ampache
-version=3.9.0
+version=4.2.3
 revision=1
 archs=noarch
```
Comment (ericonr) [inline srcpkgs/ampache/template:5]: Remove noarch.

File: srcpkgs/ampache/template [hunk 2]
```diff
@@ -11,7 +11,7 @@ maintainer="Steve Prybylski <sa.prybylx@gmail.com>"
 license="AGPL-3.0-or-later"
 homepage="http://ampache.org"
 distfiles="https://github.com/ampache/ampache/releases/download/${version}/${pkgname}-${version}_all.zip"
-checksum=2c232e080eb9ea6c77cea7b078b89e94079f5629cfe7b1de578de92c1240b892
+checksum=26e5984d5582c0bd1789626c7b07ce86ada86d64e0e5549baac625236a36e5ad
 python_version=2
```
Comment (ericonr) [inline srcpkgs/ampache/template:15]: Is it still on python 2? :/

Comment (aurieh) [reply srcpkgs/ampache/template:15]: Honestly not sure why that's specified even, I can't find any python scripts in the ampache source tree.

Comment (aurieh) [reply srcpkgs/ampache/template:15]: It's clear now: a few dependencies pulled in by ampache rely on python scripts (`php-openid` specifically), and they're almost exclusively Python 2.

Comment (ericonr) [reply srcpkgs/ampache/template:15]: I don't think it should require the line, but I don't fully understand how this part works. If you want, you can try building without the line.

Comment (aurieh) [reply srcpkgs/ampache/template:14]: It doesn't build without it (`find` broken pipe error, something `python_version` being unset), so it seems that it is necessary.

**Comment by ericonr** [comment]
@stpx would you like to give the package up for adoption?

**Pull request merged**